private-pools-example/README.md

```diff
@@ -0,0 +1,287 @@
+# Accessing private resources using Cloud Build Private Pools
+
+In this repository, we will demonstrate the below functionality of Cloud Build Private Pools:
```

Comment:
link to: https://cloud.google.com/build/docs/private-pools/private-pools-overview

Reply:
Updated.